description.xml

```diff
@@ -6,8 +6,8 @@
 		<nl>Stichting Privacy by Design</nl>
```

Comment:
Shouldn't the name be SIDN or even Yivi?

Reply:
It is the `pbdf` scheme, so it would be a bit weird to name it SIDN in my view.